ui/package.json

```diff
@@ -3,12 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "ajv": "^6.5.2",
+    "moment": "^2.22.2",
     "patternfly": "^3.52.2",
     "patternfly-react": "^2.11.1",
     "react": "^16.4.1",
+    "react-datetime": "^2.15.0",
     "react-dom": "^16.4.1",
     "react-router-dom": "^4.3.1",
-    "react-scripts": "1.1.4"
+    "react-scripts": "1.1.4",
+    "redux": "^4.0.0",
+    "table-resolver": "^4.1.1"
```

Comment:
I believe that we should not use `^` to define the deps since it could install an upper version with break changes which will affect the project.